src/components/ArrowKeyFocusManager.js

```diff
@@ -21,11 +21,15 @@ const propTypes = {
 
     /** If this value is true, then we exclude TextArea Node. */
     shouldExcludeTextAreaNodes: PropTypes.bool,
+
+    /** If this value is true, then the arrow down callback would be triggered when the max index is exceeded */
+    shouldTriggerArrowDownWhenMaxIndexExceeded: PropTypes.bool,
```

Comment:
Since the max index exceeded means we've reached the end of the list, we can call it `shouldResetIndexOnEndReached`
I'm also ok with `shouldResetFocusedIndexWhenMaxIndexExceeded`, it's long but self-explanatory

Reply:
I like `shouldResetIndexOnEndReached`, can we use that please